pkg/antctl/command_definition.go

```diff
@@ -121,8 +131,19 @@ type endpoint struct {
 	addonTransform func(reader io.Reader, single bool) (interface{}, error)
 }
 
+// FlagType represents type of a flag
+type FlagType uint8
+
+const (
+	// StringFlag represents a string flag
+	StringFlag FlagType = iota
+	// BoolFlag represents a boolean flag
+	BoolFlag
```

Comment:
Do we still need this?

Reply:
Indeed, BoolFlag is not applied by any commands now. However, I suggest keeping it since these newly-added types not just add support for "bool" flag but also it refactors the code to adapt more kinds of flags in the future. Unless there will only be string flag, I think it is worthwhile to keep these changes.

Reply:
Then we should add this the first time it is used, not assuming it can be used one day unpredictably. Some people are even dedicated to clean up functions, structs that are no longer used no matter how useful they used to be. We shouldn't do the opposite thing.

Reply:
Removed.